ext/nnstreamer/tensor_filter/tensor_filter_edgetpu.cc

```diff
@@ -151,10 +181,71 @@ tensor_filter_subplugin & edgetpu_subplugin::getEmptyInstance ()
   return *(new edgetpu_subplugin());
 }
 
+/**
+ * @brief Internal helper to get lowercase string of the given one.
+ * @param s The given std:string value
+ * @return A lowercase version of s
+ */
+std::string edgetpu_subplugin::str_tolower(std::string s) {
+  std::transform(s.begin(), s.end(), s.begin(),
+      [](unsigned char c){ return std::tolower(c); });
+
+  return s;
+}
+
+/**
+ * @brief Internal method to parse custom properties
+ * @param custom_prop The given c_str value of the 'custom' property
+ * @return A edgetpu_subplugin_device_type value for the given 'device_type'
+ *        if the value of the 'custom' property. Otherwise,
+ *        edgetpu_subplugin_device_type::USB, which is default, is returned.
+ */
+edgetpu_subplugin_device_type edgetpu_subplugin::parse_custom_prop (
+    const char *custom_prop)
+{
+  if ((!custom_prop) || (strlen (custom_prop) == 0))
+    return edgetpu_subplugin_device_type::DEAFULT;
+
+  const std::string cprop (custom_prop);
+  const std::string key ("device_type");
+  const std::size_t max_vec_len = 2;
+  std::vector<std::string> vec;
+  std::stringstream cprop_ss;
+  std::string token;
+  std::size_t pos;
+
+  pos = cprop.find_first_of (',');
+  cprop_ss = std::stringstream (cprop.substr (0, pos));
+
+  while (std::getline (cprop_ss, token, ':')) {
+    vec.push_back (token);
+    if (vec.size () > max_vec_len)
+      break;
+  }
+
+  if (edgetpu_subplugin::str_tolower (vec[0]) != key)
+    return edgetpu_subplugin_device_type::USB;
```

Comment:
If DEFAULT is being used, then this and at line 240 would also be DEFAULT.

Reply:
Right. Thanks.